mlrun/model_monitoring/db/stores/__init__.py

```diff
@@ -16,60 +16,61 @@
 
 import enum
 import typing
+import warnings
 
 import mlrun.common.schemas.secret
 import mlrun.errors
 
-from .model_endpoint_store import ModelEndpointStore
+from .base import StoreBase
 
 
-class ModelEndpointStoreType(enum.Enum):
-    """Enum class to handle the different store type values for saving a model endpoint record."""
+class ObjectStoreType(enum.Enum):
+    """Enum class to handle the different store type values for saving model monitoring records."""
 
     v3io_nosql = "v3io-nosql"
     SQL = "sql"
 
-    def to_endpoint_store(
+    def to_object_store(
         self,
         project: str,
         access_key: str = None,
-        endpoint_store_connection: str = None,
+        store_connection: str = None,
```

Comment:
As we talked, the user should access the store object using the `get_store_object` that doesn't support this store_connection param. The reason I left it here it's only because we use through the tests to generate a dummy object directly. Anyway, we can remove it and adjust the tests but we can do it in a different PR.